html2/tokenizer.cpp

```diff
@@ -1,15 +1,19 @@
-// SPDX-FileCopyrightText: 2021 Robin Lindén <dev@robinlinden.eu>
+// SPDX-FileCopyrightText: 2021-2022 Robin Lindén <dev@robinlinden.eu>
 //
 // SPDX-License-Identifier: BSD-2-Clause
 
 #include "html2/tokenizer.h"
 
+#include "html2/character_reference.h"
 #include "util/string.h"
 
 #include <spdlog/spdlog.h>
 
+#include <algorithm>
```

Comment:
Is this include needed?

Reply:
Nope! Forgot to remove it when simplifying my code. Good catch, removed!